checkEligibility.js

```diff
@@ -31,7 +31,13 @@ async function patronCanPlaceTestHold (patronId, firstAttempt = true) {
         return true
       }
     } else {
-      patronHoldsPossible = e.response.data.description === 'XCirc error : Bib record cannot be loaded'
+      const { description, name } = e.response.data
+      patronHoldsPossible = (
+        description.includes('Bib record cannot be loaded')
+      ) && (
+        name === 'XCirc error' || description.includes('XCirc error')
+      )
```

Comment:
So in Sierra <6.1, the error manifests as: `{ description: "XCirc error : Bib record cannot..."}`.. and in Sierra 6.1, the error manifests as `{ name: "XCirc error", description: "Bib record cannot..."}`? Can you add a comment so we don't forget what we're targeting here?

Reply:
Added!